CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Contributing to azure-devops-dependency-visualizer 
+
+Contributions to azure-devops-dependency-visualizer are welcome from all!
+
+azure-devops-dependency-visualizer is managed via [git](https://git-scm.com), with the canonical upstream
+repository hosted on [GitHub](https://github.com/ni/azure-devops-dependency-visualizer/).
+
+azure-devops-dependency-visualizer follows a pull-request model for development.  If you wish to
+contribute, you will need to create a GitHub account, fork this project, push a
+branch with your changes to your project, and then submit a pull request.
+
+See [GitHub's official documentation](https://help.github.com/articles/using-pull-requests/) for more details.
+
+# Getting Started
+
+Install Node.js and run 'npm install' from the src directory in order to install dependencies.  See [Microsoft's documentation](https://docs.microsoft.com/en-us/azure/devops/extend/?toc=%2Fazure%2Fdevops%2Fextend%2Ftoc.json&bc=%2Fazure%2Fdevops%2Fextend%2Fbreadcrumb%2Ftoc.json&view=azure-devops) for information on developing and publishing extensions.
+
+# Before Posting a Pull-Request
+
+1. Document your changes.  If adding new functionality, document it in the src/content/overview.md file that will be displayed on the extension's Marketplace page. Add screenshots as appropriate.
+2. TEST!  See the 'Testing' section, below.
+3. Increment the version as appropriate. Versioning is less critical on an extension like this but try to stick to bug fixes => patch; incremental features => minor; bigger features => major.
+
+## Testing
+
+Prior to posting a pull-request, follow the steps below.
+
+1. Modify the vss-extension.json file to set the extension to private.
+2. Publish the extension to the [Marketplace](https://marketplace.visualstudio.com/) under your own account.
+3. Share the published extension with an Azure DevOps organization where you have install permissions and install the extension.
+4. Thoroughly test your changes. It's a good idea to take notes on what you tested and use this as a guide when you...
+5. ...update the 'Test Cases' section, below. If introducing a lot of new functionality, consider migrating the 'Test Cases' section to a separate file and linking it from this document.
+6. Exercise the existing test cases and ensure there are no regressions.
+7. In the vss-extension.json file, change the extension back to public and reset the version appropriately if multiple increments were required during deployment for testing purposes.
+
+### Test Cases
+
```

Comment:
Any plans to automate any of these? The standard toolchain of Karma/Jasmine for unit tests is pretty easy to set up and automate in TravisCI or Azure Pipelines.